src/librustdoc/html/static/js/main.js

```diff
@@ -1620,7 +1621,7 @@ function preLoadCss(cssUrl) {
             ["↓", "Move down in search results"],
             ["← / →", "Switch result tab (when results focused)"],
             ["&#9166;", "Go to active search result"],
-            ["+", "Expand all sections"],
+            ["+ / =", "Expand all sections"],
```

Comment:
We generally avoid adding new shortcuts to not overwrite existing browser/user's shortcuts.

Reply:
the current `+` shortcut is not just only `+`, it's really `shift`+`=`.

Reply:
User press `-` (no `shift` needed), it works. But if he press `+` (without `shift`), it not works, this is a bit weird, or not convenient.

Reply:
That's not the case for me on firefox, what browser and keyboard layout are you using?

Reply:
Very strange, that's the same here, you don't happen to have any extensions or userscripts that could be changing that behavior, do you?

What's your exact browser version?

Reply:
Only adblock. Version 143.0.3.

Reply:
Ah ok, I understand the issue now. It's because I use the numpad. Well I guess `=` is fine then.